### DISCUSSION TITLE
Überstundenauszug

### DISCUSSION THREAD

**Discussion by pkoevesdi** [kimai/Paid-plugins#200] [General]
I tried to find it, but got not really lucky: From time to time our management wants an overview of the plus/minus hours (german: Überstunden) of all employees. What's a best practice or where is the function to get such?

I see this information under "working times" (`/contract`), but only for one user at a time. We'd like to have a table over all users. Is this possible, except via database query?

**Comment by kevinpapst** [comment]
No, not available as of now. But its on the roadmap of the WorkContract plugin.
